### DISCUSSION TITLE
hi, will this work for lenovo legion go for zorin os? ubuntu distro?

### DISCUSSION THREAD

**Discussion by DOOMMARINE117** [BartoszCichecki/LenovoLegionToolkit#1412] [Q&A] [answered]
Hi, will this work for lenovo legion go for zorin os ubuntu distro?

**Comment by BartoszCichecki** [comment, marked as answer]
No. It's clearly stated that it's for **laptops** and it runs on Windows.

**Reply by DOOMMARINE117** [reply]
> No. It's clearly stated that it's for **laptops** and it runs on Windows.

thanks.
